Introduce new method GetIncludeDir() in GeneralUtilities.

petterreinholdtsen opened the pull request:
This make it easier to create a Debian package, as the include
directory in the package is /usr/share/creepy/include, which is
not in the current working directory.

petterreinholdtsen committed:
Introduce new method GetIncludeDir() in GeneralUtilities.
This make it easier to create a Debian package, as the include
directory in the package is /usr/share/creepy/include, which is
not in the current working directory.

jkakavas committed:
Bringing the source code to the latest version to continue development in this fork

jkakavas committed:
Implemented pull requests from the old repository. Fixed flickr to use oauth

jkakavas committed:
latest versions of all plugins

jkakavas commented:
hi, I have merged this and all the other pull requests you have sent in https://github.com/jkakavas/creepy. Thank you very much for your effort and your time in designing and implementing the changes so that creepy can work in debian

jkakavas committed:
removed PyCharm project files and added them to gitignore

jkakavas committed:
Modifications in plugins

jkakavas committed:
minor changes

jkakavas committed:
Final changes for v1.3, added inno setup script under version control

jkakavas committed:
Changed map.html to load marker icons from github instead of locally

jkakavas committed:
Fixed version

Commit:
Add new png and ico files

Commit:
Added a title to the plugin wizard window

Commit:
Fixed a bug in flickr causing no locations to be found if photos results were more than one page

Commit:
Fixed version and name in the about dialog

jkakavas committed:
Update README.md

Commit:
Fixed bug with heatmap containing locations from previously analyzed projects

Commit:
Merge branch 'master' of github.com:jkakavas/creepy

LoneStarMac committed:
Update creepy_windows_installer.iss

This will properly set the HOMEPATH environment variable in Windows.

jkakavas committed:
Merge pull request #13 from LoneStarMac/patch-1

Update creepy_windows_installer.iss

Commit:
Added dockable window that shows log output in the main window for better visibility

Commit:
Added new wizard for place based projects

Commit:
Added clustering capabilities to the locations visualization

Commit:
Added functionality for place based project

Commit:
Added functionality to check API Calls rate limits for plugins that support it

Commit:
Added a window to show logs in the main interface

Commit:
Changes in the User Interface

Commit:
Update Qt resource files

Commit:
Updated models

Commit:
Updated components

Commit:
Added search option for address in the wizard for Place based project

Commit:
Updated plugins

Commit:
Updated the main script to the new version

Commit:
Merge branch 'master' of github.com:jkakavas/creepy

Commit:
Minor changes and bugfixes from initial testing of v1.4

Commit:
Added a button in the instagram config wizard to open the link automatically in the browser instead for instrcuting the user to copy paste it

Commit:
Updated README

Commit:
removed unecessary explicit setting of cacerts.txt for Linux

Commit:
Fixed version numbers in the UI

Commit:
Import pytz again which should not have been removed as it is required by the FilterLocationsByDate. Spotted by @atekoa on #21

petterreinholdtsen committed:
Introduce getProjectsDir() and getTempDir().

This make it easier to redirect the projects and temp directories
when running creepy outside its build directory.

petterreinholdtsen committed:
Fix typo in twitter plugin.

jkakavas committed:
Merge pull request #24 from petterreinholdtsen/redirect-temp-project-dir

Introduce getProjectsDir() and getTempDir().

jkakavas committed:
Merge pull request #25 from petterreinholdtsen/plugin-twitter-typo

Fix typo in twitter plugin.

petterreinholdtsen committed:
Merge branch 'master' into redirect-include-dir

Conflicts:
	creepy/CreepyMain.py
	creepy/models/ProjectWizardPluginListModel.py
	creepy/utilities/GeneralUtilities.py
